gdraw/gtextfield.c

```diff
@@ -1746,22 +1746,30 @@ return( true );
 	if ( gt->pressed==NULL )
 	    gt->pressed = GDrawRequestTimer(gt->g.base,200,100,NULL);
 	if ( gt->sel_start > u_strlen( gt->text ))	/* Ok to have selection at end, but beyond is an error */
-	    fprintf( stderr, "About to crash\n" );
-	_ggadget_redraw(g);
-return( true );
+        _ggadget_redraw(g);
```

Comment:
The comment has a point, even if it might be better as an informed logged error. If you have a reproducible case that enters this code block it would be better to fix it. It would also be better to know why it doesn't cause a crash.

Reply:
I do have a reproducible case, it's #3620